pom.xml

```diff
@@ -2649,6 +2649,13 @@
     </profile>
 
     <profile>
+      <id>kubernetes</id>
```

Comment:
this new profile is not use in build script, shall we add this so that the new contributions in this PR will be built and tested?

Reply:
We should also change the sbt file to make it work using sbt.

Reply:
Updated. PTAL.